raft/tracker/progress.go

```diff
@@ -48,7 +45,12 @@ type Progress struct {
 	// RecentActive can be reset to false after an election timeout.
 	RecentActive bool
 
-	// inflights is a sliding window for the inflight messages.
+	// RecentlyContacted is used in StateProbe.
+	// When RecentlyContacted is true, raft should pause sending replication
+	// message to this peer until RecentlyContacted is reset. See IsPaused().
+	RecentlyContacted bool
```

Comment:
Good call, I like ProbeSent because it indicates what happens and also makes it not surprising that this is reset periodically (to send another probe).